src/routes/routeConstants.ts

```diff
@@ -56,4 +56,4 @@ export const LANDING_PAGE_ROUTE = '/';
 export const TX_SEARCH_PARAM = 'tx';
 export const TEAM_SEARCH_PARAM = 'team';
 
-export const HOST = window.location.host;
+export const HOST = process.env.HOST || 'localhost:9091';
```

Comment:
This should be removed entirely, and just use `process.env.HOST` directly, not set inside a `const` and without a fallback